Sources/MapboxNavigation/InstructionsBannerView.swift

```diff
@@ -6,7 +6,7 @@ import MapboxDirections
 /**
  `InstructionsBannerViewDelegate` provides methods for reacting to user interactions in `InstructionsBannerView`.
  */
-public protocol InstructionsBannerViewDelegate: AnyObject, UnimplementedLogging {
+public protocol InstructionsBannerViewDelegate: VisualInstructionDelegate {
```

Comment:
This protocol should still conform to `AnyObject` and `UnimplementedLogging` for backwards compatibility. Also, `UnimplementedLogging` is useful for highlighting missing delegate method implementations.

Reply:
But because `VisualInstructionDelegate` already conforms to `AnyObject` and `UnimplementedLogging`, there's a redundant declaration of the two protocols.